CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Crash when starting on some X11 systems
 - Font size resetting when Alacritty is moved between screens
 - Limited payload length in clipboard escape (used for Tmux copy/paste)
+- Force Application symlink when packaging for macOS
```

Comment:
Could we mention which failure we've fixed, rather than how we fixed it?

Something like this maybe:

```suggestion
- Repeated builds failing on macOS due to preexisting /Application symlink
```

Reply:
Should we note that it's only related to `make`, something like "Repeated builds with `make` failing..."

Reply:
"Repeated `make` builds failing..."? Maybe even "Successive (or sequential) `make` builds failing..." since it appears the failure is linked (ha) to not building cleanly.

Reply:
Great suggestions from both of you, thanks!

I like "Successive `make` builds failing...".

Reply:
That's the word I was going for, but it was slipping my mind.

My vote is for "Consecutive `make` builds failing...".

Reply:
Teamwork! :smile:

Reply:
```suggestion
- Consecutive builds failing on macOS due to preexisting `/Application` symlink
```